cli/medperf/commands/mlcube/submit.py

```diff
@@ -110,9 +128,12 @@ def todict(self):
             "state": "OPERATION",
             "is_valid": True,
         }
+        if self.image_file:
+            dict["image_tarball_url"] = self.image_file
+            dict["image_tarball_hash"] = self.image_tarball_hash
```

Comment:
At some point, we should change "image_tarball" to just "image" throughout this class for clarify.